test/run-unittest.d

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env dub
+/+dub.sdl:
+	name: run_unittest
+	targetName: run-unittest
++/
+module run_unittest;
+
+/// Name of the log file
+enum logFile = "test.log";
+
+/// has true if some test fails
+bool any_errors;
+
+/// prints (non error) message to standard output and log file
+void log(Args...)(Args args)
+	if (Args.length)
+{
+	import std.format : format;
+	import std.stdio : File, stdout, writeln;
+
+	version(Windows) const str = format("[INFO] " ~ args[0], args[1..$]);
+	else const str = format("\033[0;32m[INFO] " ~ args[0] ~ "\033[0m", args[1..$]).format(args[1..$]);
+	writeln(str);
+	stdout.flush;
+	File(logFile, "a").writeln(str);
```

Comment:
Oh I just realized that the executed shell scripts also write to the same file, right? If that is the reason, then I take this comment back. You may want to leave a note in the source about this.

Reply:
Yes, it is sub-optimal and no, it is not intended to be written to by executed shell scripts. I just try to repeat existing shell script and make code review easier, all enhancements later.

Reply:
You are right, some scripts write to the same file. I compare output of both versions again and found this difference. It is surprise for me because I checked that last week and it was ok.

Reply:
I got it. The shell script searches the log file in the directory where the script is placed and the d script searches where it is run from.